templates/components/common/body.html

```diff
@@ -1,4 +1,4 @@
-<main class="body" id='main-content' role='main' data-currency-code="{{currency_selector.active_currency_code}}">
+<main class="body" id="main-content" role="main" data-currency-code="{{currency_selector.active_currency_code}}">
```

Comment:
Do we really need to duplicate main tag and main role?

Reply:
it's a [recommendation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) on MDN to better support IE11.